src/style.css

```
@@ -1,3 +1,78 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+:root {
+  --light-blue: rgb(0, 172, 220);
+  --white: rgb(255, 255, 255);
+}
+
+* {
```

These classes should be in the `html` selector
```suggestion
html {
```